src/components/app-config.tsx

```diff
@@ -1,50 +1,21 @@
-import React, { memo } from 'react';
+import React from 'react';
 import { RecoilRoot } from 'recoil';
 import { AppContainer } from '@components/app-container';
-import { SWRConfig } from 'swr';
-import { DefaultOptions, QueryClient, QueryClientProvider } from 'react-query';
-import { Hydrate } from 'react-query/hydration';
 import { CacheProvider } from '@emotion/react';
 import { cache } from '@emotion/css';
-import { DEFAULT_POLLING_INTERVAL } from '@common/constants';
-
-const config: DefaultOptions['queries'] = {
-  refetchInterval: DEFAULT_POLLING_INTERVAL,
-  keepPreviousData: true,
-  notifyOnChangeProps: ['data'],
-};
-
-const queryClient = new QueryClient({
-  defaultOptions: {
-    queries: config,
-  },
-});
 
 interface AppConfigProps {
   isHome?: boolean;
   fullWidth?: boolean;
   dehydratedState?: any;
 }
 
-export const AppConfig: React.FC<AppConfigProps> = memo(
-  ({ children, isHome, fullWidth, dehydratedState }) => (
-    <SWRConfig
-      value={{
-        refreshInterval: DEFAULT_POLLING_INTERVAL,
-        suspense: false,
-      }}
-    >
-      <RecoilRoot>
-        <QueryClientProvider client={queryClient}>
-          <Hydrate state={dehydratedState && eval(dehydratedState)}>
-            <CacheProvider value={cache}>
-              <AppContainer isHome={isHome} fullWidth={fullWidth}>
-                {children}
-              </AppContainer>
-            </CacheProvider>
-          </Hydrate>
-        </QueryClientProvider>
-      </RecoilRoot>
-    </SWRConfig>
-  )
+export const AppConfig: React.FC<AppConfigProps> = ({ children, isHome, fullWidth }) => (
+  <RecoilRoot>
```

Comment:
I see a bit of recoil is being left, I thought we were totally removing Recoil?

Reply:
I did not remove recoil from the sandbox -- that is a bit bigger and this PR is already larger than I hoped for. I'll work on that in a follow up PR

Reply:
That makes sense, got it, thx!